_posts/2019/2019-02-24-secure-your-staging-environment-with-basic-auth.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Secure Your Rails Staging Environment with HTTP Basic Authentication
+description: Opening up your development environment is a bad idea
+layout: article
+category: ruby
+image:
+  base: '2019/secure-your-staging-environment-with-basic-auth'
+  alt: 'Yellow padlock'
+  source: "https://unsplash.com/photos/0Yiy0XajJHQ"
+  credit: "Chris Panas"
+---
+
+A sensible approach to testing features before you ship them to customers is to use a staging environment that closely resembles, your live, customer-facing, production application.
+
+It’s important to make sure that only people inside your organisation can use this testing environment, it might contain broken features (hopefully not!) or
```

Comment:
*environment, since it might contain broken features.

Or..? Yes I was thinking you needed another reason here to! :-D (Maybe security concerns?)